pkg/services/ngalert/state/manager.go

```diff
@@ -237,39 +237,39 @@ func translateInstanceState(state ngModels.InstanceStateType) eval.State {
 	}
 }
 
-func (st *Manager) createAlertAnnotation(ctx context.Context, alertRule *ngModels.AlertRule, labels data.Labels, evaluatedAt time.Time, state eval.State, previousState eval.State) {
-	st.log.Debug("alert state changed creating annotation", "alertRuleUID", alertRule.UID, "newState", state.String(), "oldState", previousState.String())
+func (st *Manager) annotateState(ctx context.Context, r *ngModels.AlertRule, instanceLabels data.Labels, evaluatedAt time.Time, state eval.State, previousState eval.State) {
```

Comment:
What do you think about keeping `alertRule` instead of renaming it to `r`? I really like when the code is "readable".